sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Azure.AI.TextAnalytics.Tests
 {
-    [Ignore("Not yet implemented")]
+    [ClientTestFixture(TextAnalyticsClientOptions.ServiceVersion.V2022_03_01_Preview)]
```

Comment:
For now, I'm enabling testing for the latest version.
I tried enabling test for all versions and the world exploded :) so I need more time to figure out what happened and fix that